documentation/users-guide/source/reference.rst

```diff
@@ -66,6 +66,20 @@ Suites, Tests, and Benchmarks
    "crashed". Other than this, and some differences in how the results
    are displayed, benchmarks are the same as tests.
 
+.. function:: benchmark-repeat
+
+   Repeatedly execute a block of code, recording profiling information for each
+   execution.
+
+   :signature: benchmark-repeat (#key *iterations* = 1) *body* end
+   :parameter iterations: Number of times to execute *body*.
+
+   Results for benchmarks that call benchmark-repeat display the min, max,
+   mean, and median run times across all iterations.
```

Comment:
"It may be necessary to use the `full` report to display detailed benchmark statistics."

Reply:
done